integrationTests/p2p/pubsub/peerReceivingMessages_test.go

```diff
@@ -20,8 +20,8 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-var durationBootstrapingTime = time.Duration(time.Second * 2)
-var durationTest = time.Duration(time.Second * 30)
+var durationBootstrapingTime = 2 * time.Second
```

Comment:
Remove createMessenger() and modify and use CreateMessengerWithKadDht() from testInitializer.

Reply:
Extract waitForBootstrapAndShowConnected() in testInitializer and use it in all of the p2p integration tests

Reply:
Use GetConnectableAddress() fro testInitializer instead of chooseNonCircuitAddress()

Reply:
Resolve warnings.

Reply:
done